src/components/TheMap.vue

```diff
@@ -26,8 +26,48 @@ export default {
   watch: {
     checkedTrenchesItems: function () {
       this.layer.clearLayers();
+      this.refreshMap();
+    },
+  },
+  mounted() {
+    this.map = L.map("mapContainer", {
+      attributionControl: false,
+      zoomControl: true,
+      //   note that Leaflet supports very few coordinate systems: CRS:3857, CRS:3395 and CRS:4326 see also L.CRS.Simple.
+    });
+
+    L.tileLayer("http://{s}.tile.osm.org/{z}/{x}/{y}.png", {
+      maxZoom: 25,
+      maxNativeZoom: 19,
+      attribution:
+        '&copy; <a href="http://osm.org/copyright">OpenStreetMap</a> contributors',
+    }).addTo(this.map);
+
+    // WMS layer from Dipylon. To be used in dev only for not overload their server
+    // L.tileLayer
+    //   .wms("http://116.202.128.162:83/geoserver/wms?SERVICE=WMS?", {
+    //     layers: "amarynthos:AMA22_complete_GGRS87",
+    //     attribution: "ESAG",
+    //     transparent: true,
+    //     maxZoom: 25,
+    //   })
+    //   .addTo(this.map);
+
+    this.refreshMap();
+  },
+
+  methods: {
+    onEachFeature(feature, layer) {
```

Comment:
Ah, le ` onEachFeature: this.onEachFeature,`, plus bas, m'a un peu embrouillé. 

C'est toujours pas tres clair, dans vue, où definir une fonction qui ne vas être utilisé qu'une fois dans une autre fonction. Je connais pas de bonne solution